azkaban-webserver/src/test/expected/project-side-bar.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+
+          <div class="well" id="project-sidebar">
+            <h3>test_project</h3>
+            <p class="editable" id="project-description">&lt;script&gt;window.echo(&quot;hacked&quot;)&lt;/script&gt;</p>
```

Comment:
What do you mean?
This file is used as a "plain text" to verify the test output.